app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const {getTopics } = require('./controllers/topics.controller.js')
+
+
+// Set up. 
+const app = express()
+app.use(express.json())
```

Comment:
Going to be picky, do you need this yet?

Reply:
No, but I'm pretty sure I'm going to.